src/main/java/org/algorithms/ValidAnagram.java

```diff
@@ -0,0 +1,30 @@
+package org.algorithms;
+
+import java.util.Arrays;
+import java.util.logging.Logger;
+
+public class ValidAnagram {
+  private static final Logger logger = Logger.getLogger(ValidAnagram.class.getName());
+
+  public boolean isAnagram(String s, String t) {
+    // Convert strings to Character Arrays
+    char[] sArray = s.toLowerCase().toCharArray();
+    char[] tArray = t.toLowerCase().toCharArray();
+
+    // Sort the arrays
+    Arrays.sort(sArray);
+    Arrays.sort(tArray);
+
+    return Arrays.equals(sArray, tArray);
```

Comment:
Don't use Arrays.sort() and Arrays.equals(). Think if it is possible without any sorting.